SwiftUIDemoAppUITests/Tests/DeeplinkTests.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  DeeplinkTests.swift
+//  SwiftUIDemoAppUITests
+//
+//  Created by Ilias Pavlidakis on 25/9/23.
+//
+
+import XCTest
+
+final class DeeplinkTests: StreamTestCase {
```

Comment:
Awesome 🚀  When are these tests being run? Should we put them as part of the checks or in a test plan?

Reply:
They are part of the UI Tests so i believe they should be running as part of the Release. Not sure about the PR checks though. Maybe @testableapple can provide more insight to this one.

Reply:
It will be executed with the other tests